include/ibamr/IIMethod.h

```diff
@@ -168,6 +168,22 @@ class IIMethod : public IBStrategy
         void* ctx;
     };
 
+/*!
+     * Returns either the current configuration (at possibly different times in the timestep)
```

Comment:
This comment needs some more information -- specifically, what determines which configuration is returned?